### DISCUSSION TITLE
A web error

### DISCUSSION THREAD

**Discussion by Adrypp** [flet-dev/flet#2609] [Q&A]
### Question

hii everyone i need some help. It turns out that using flet publish I have created a website but every time I run the python -m http.server 8000 it does not create the server because it runs into the file Python.js

I am new with this frame and language 

What happen?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by 09u2h4n** [comment]
Well if you created with `flet publish` it takes time to load for example a basic "Hello world program" takes around ~30 seconds to load. Maybe you don't wait enough time to load 🤔.

Can you send the console log (Console is in the google dev tools) of the web app or code itself?

**Reply by 09u2h4n** [reply]
Maybe try `flet build web app_folder/` instead of `flet publish`
